packages/commonwealth/server/migrations/20241211135155-add-unsubscribe-uuid-to-user.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn('Users', 'unsubscribe_uuid', {
```

Comment:
can we add service name to `unsubscribe_uuid`? ex: `knock_unsubscribe_uuid` or similar?

Reply:
I don't think this is necessary. The UUID is not service-specific. We would need this value even if we used another email provider.